alibuild_helpers/build.py

```diff
@@ -451,6 +452,11 @@ def doBuild(args, parser):
              'Maybe you need to "cd" to the right directory or '
              'you forgot to run "aliBuild init"?' % args.configDir)
 
+  if (not args.no_remote_store and
```

Comment:
The change itself is fine for me. What happens on platforms where we prefer the system tools, like on macOS?

Could you add a test which exercises this part of the code?